loopout.go

```diff
@@ -31,7 +31,7 @@ var (
 	// confirmation target.
 	//
 	// TODO(wilmer): tune?
-	DefaultSweepConfTargetDelta int32 = DefaultSweepConfTarget * 2
+	DefaultSweepConfTargetDelta = DefaultSweepConfTarget * 2
```

Comment:
Shouldn't all these be `const` really?

Reply:
Yes, they could be const. There are some in `loopin.go` too. I just fixed this because my IDE complained. Will leave further changes out of this PR.